modules/hosted-control-planes-troubleshooting.adoc

```diff
@@ -41,56 +41,10 @@ Although the output does not contain any secret objects from the cluster, it can
 +
 [source,terminal]
 ----
-$ hypershift dump cluster \
-    --name <hosted_cluster_name> \// <1>
-    --namespace <hosted_cluster_namespace> \ <2>
-    --dump-guest-cluster \
-    --artifact-dir clusterDump-<hosted_cluster_namespace>-<hosted_cluster_name>
+$ oc adm must-gather --image=registry.redhat.io/multicluster-engine/must-gather-rhel8:v2.x /usr/bin/gather hosted-cluster-namespace=HOSTEDCLUSTERNAMESPACE hosted-cluster-name=HOSTEDCLUSTERNAME --dest-dir=NAME ; tar -cvzf NAME.tgz NAME
```

Comment:
`registry.redhat.io/multicluster-engine/must-gather-rhel8:v2.x` will this image change depending on MCE version? 

Is this a public image or will they need to use a pull secret with this command?

Reply:
Yes, the version of this image changes depending on MCE version and requires a pull secret.

Reply:
@rokej I don't think we mention anything about a pull secret in these docs. Do I need to add another step for the user to supply the pull secret?

Reply:
I think we would also need to add something about replace the x in `2.x` with their version of MCE.

Reply:
@bryan-cox @rokej I added a line to instruct the user to replace 2.x with their version of MCE. Let me know what you think when you can. Thanks!